uaclient/tests/test_ua_timer.py

```diff
@@ -136,3 +136,29 @@ def test_run_job_ignores_late_next_run(self, FakeConfig):
 
         assert "last_run" not in cfg.read_cache("jobs-status")["day_job"]
         assert 0 == m_job_func.call_count
+
+
+class TestMeteringTimedJob:
+    @pytest.mark.parametrize(
+        "activity_ping_interval_value,config_value,expected_value",
+        ((None, 100, 100), (20, 80, 80), (1000, 80, 1000)),
+    )
+    @mock.patch("lib.timer.TimedJob.run_interval_seconds")
+    def test_metering_run_interval_seconds(
+        self,
+        m_run_interval_seconds,
+        activity_ping_interval_value,
+        config_value,
+        expected_value,
+    ):
+        m_run_interval_seconds.return_value = config_value
+        m_cfg = mock.MagicMock()
```

Comment:
You could mock the config here with FakeConfig, and pass a dict with the properties (`activity_ping_interval` and `metering_timer`) when creating the instance

Reply:
I think I am missing something here, the `activity_ping_interval` is config property. I don't think we can directly override it with `cfg_overrides`.

Reply:
Indeed... You would need to pass the whole structure (machine_token -> activityInfo -> activity_ping_interval) to cfg_overrides. Feel free to ignore this